### DISCUSSION TITLE
共同创造价值 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#124] [Comments]
# 共同创造价值 | 夜天之书

如何吸引开源开发人员参与项目？如何让他们留下来，成为项目共同体的一部分？这是两个做开源运营必须回答的问题。 我对这两个问题的回答，简而言之是和开源参与者共同创造价值，使得开源项目和开源共同体能够回答潜在参与者的两个事关去留的灵魂提问。  我能为你做什么？ 我应该怎么做到？  从共同创造价值的角度出发，通过开源运营回答参与者可以做什么的问题，只有可做的事情是令人兴奋的价值创造，才有可能触发潜在的参与

https://www.tisonkun.org/2022/02/10/value-creation/

**Comment by ZoomQuiet** [comment]
> "不要问社区为我作了什么, 先问我为社区作了什么"...

哈, 其实, 如何成为 Hacker 那篇老文中已经明确指出什么是有效贡献有20年了,
但是, 慢慢就会发现, 每一种贡献都有进一步细节的区别, 以及应对最佳模式,

细节中有魔鬼...如果是全义务志愿者组成的自由软件社区还好,
商业团队触发的开源社区才复杂,
因为, 核心成员本身是多重身份, 而且其职员身份权重更大,
也就是说, 企业这个老大哥将无时不刻影响着所有成员,
此时, OSPO 的价值才体现出来吧,
其实, 就是: 舞台总监, 监察和管理所有企业开源成员的言行, 要兼容内外期待;

反而外部自由成员不受这种心理压力, 可以自由奔放的以好奇心为驱动来贡献;

问题就在响应上, 一但响应不及时, 就无法收尾了...
